metadata/classes.json

```diff
@@ -3775,4 +3831,4 @@
 		"wiggleVIbratoLargestSlower",
 		"wiggleVibratoLargestSlowest"
 	]
-}
+}
```

Comment:
Not sure this change is intentional and/or wanted?

Reply:
Sorry, could you be a bit more specific in your comment? I feel like I'm being a bit slow, but are you questioning whether or not the closing brace should be there?

Reply:
Hmm, it looks different here/now, it did indicate a missing line ending

Reply:
I'm talking about this:
![foo](https://user-images.githubusercontent.com/1786669/98807709-58f37800-241b-11eb-8508-cfd9dbbc8c2e.png)